xonsh/history/main.py

```diff
@@ -26,7 +27,9 @@ def construct_history(**kwargs):
     """Construct the history backend object."""
     env = builtins.__xonsh_env__
     backend = env.get('XONSH_HISTORY_BACKEND')
```

Comment:
And we even could make `XONSH_HISTORY_BACKEND` also receive a class instance. But that may go too far?